spec/requests/v1/tenant_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Hyku::API::V1::TenantController, type: :request, clean: true, multitenant: true do
+  let(:account) { create(:account, name: 'test') }
+  let(:json_response) { JSON.parse(response.body) }
+
+  before do
+    WebMock.disable!
+    Apartment::Tenant.create(account.tenant)
+    Apartment::Tenant.switch(account.tenant) { Site.update(account: account) }
+  end
+
+  after do
+    WebMock.enable!
+    Apartment::Tenant.drop(account.tenant)
+  end
+
+  describe "/tenant/:id" do
+    let(:work_types) { ["Article", "Book", "ThesisOrDissertation", "BookChapter"] }
```

Comment:
I think I'm missing something.  Where does this list get set on the account?

Reply:
Hi Chris. They were defined in the Account Factory. I've moved them into the spec to make them more self documenting though and added an extra spec to prove that they are outputting the result of the account settings.